packages/mdc-card/_mixins.scss

```diff
@@ -301,7 +301,7 @@
   }
 
   @include mdc-feature-targets($feat-color) {
-    border-color: mdc-theme-prop-value($color);
+    @include mdc-theme-prop(border-color, $color);
```

Comment:
You can revert this now since `theme-prop-value` always returns `fallback` value. We can modify this in all other places in later PRs.